guides/release/tutorial/ember-data.md

```diff
@@ -25,29 +25,31 @@ When we open the model file, we can see a blank class extending [`DS.Model`](htt
 
 ```javascript {data-filename=app/models/rental.js}
 import DS from 'ember-data';
+const { Model } = DS;
 
-export default DS.Model.extend({
+export default class RentalModel extends Model {
 
-});
+}
 ```
 
 Let's define the structure of a rental object using the same attributes for our rental that we [previously used](../model-hook/) in our hard-coded array of JavaScript objects -
 _title_, _owner_, _city_, _category_, _image_, _bedrooms_ and _description_.
 Define attributes by giving them the result of the function [`DS.attr()`](https://www.emberjs.com/api/ember-data/release/classes/DS/methods/attr?anchor=attr).
 For more information on Ember Data Attributes, read the section called [Defining Attributes](../../models/defining-models/#toc_defining-attributes) in the guides.
 
-```javascript {data-filename="app/models/rental.js" data-diff="+4,+5,+6,+7,+8,+9,+10"}
+```javascript {data-filename="app/models/rental.js" data-diff="+2,+5,+6,+7,+8,+9,+10,+11"}
 import DS from 'ember-data';
-
-export default DS.Model.extend({
-  title: DS.attr(),
-  owner: DS.attr(),
-  city: DS.attr(),
-  category: DS.attr(),
-  image: DS.attr(),
-  bedrooms: DS.attr(),
-  description: DS.attr()
-});
+const { Model, attr } = DS;
+
+export default class RentalModel extends Model {
+  @attr title;
+  @attr owner;
```

Comment:
Ember-data decorators weren't listed in https://github.com/emberjs/rfcs/pull/408. I pinged @igort to confirm that this is (a) supported and (b) something we want to steer people towards.

Reply:
cc: @mike-north thanks for fact checking this. I had opened another issue related to this. pzuraq gave some insight in this comment: https://github.com/ember-learn/guides-source/issues/554#issuecomment-469369178

Reply:
I double checked with @runspired and it's supported and people should use it

Reply:
Oh, one thing that does need to change here though, I'm pretty sure you'll still have to do `@attr()` here, since these are functions that _return_ decorators, and are not decorators themselves. Bit of a pain, but this will be the case until the ecosystem updates to support paren-less invocations.